src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -18,7 +21,7 @@ public Ladder(DirectionStrategy directionStrategy, Height height, int countOfPer
     }
 
     private static Line createLine(DirectionStrategy directionStrategy, int countOfPerson) {
```

Comment:
```suggestion
    private static Line createLine(DirectionStrategy directionStrategy, int columnSize) {
```

관점에 따라 다르겠지만, 
호출하는쪽에서의 이름이 아닌  객체의 입장을 고려해서 일반화된 이름을 사용할것 같습니다.
그렇게해야, 다른 개발자가 보더라도 흐름을 금방이해할수 있을것 같아서요~ 🤔